src/instructions/onboarding_exam/EntranceOnboardingExamInstructions.jsx

```diff
@@ -89,7 +93,7 @@ const EntranceOnboardingExamInstructions = () => {
         <Button
           data-testid="start-exam-button"
           variant="primary"
-          onClick={createProctoredExamAttempt}
+          onClick={() => dispatch(createProctoredExamAttempt())}
```

Comment:
[question] For the codecov, I think you'd mentioned coming back to this as a larger effort to clean up / update testing, right? So for now, not worrying about codecov?

Reply:
Yeah, I think so. The tests in here are kind of difficult to work with. I think we want to get all of our code changes into the feature branch, and then we'll do a mini bug bash and add any missing tests. Does that sound good to you?